launcher/test-framework/org.jboss.tools.common.launcher.reddeer/src/org/jboss/tools/common/launcher/reddeer/wizards/JBossCentralProjectWizardLauncherApplication.java

```diff
@@ -0,0 +1,233 @@
+ * and is available at http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributor:
+ *     Red Hat, Inc. - initial API and implementation
+ ******************************************************************************/
+package org.jboss.tools.common.launcher.reddeer.wizards;
+
+import org.eclipse.reddeer.common.logging.Logger;
+import org.eclipse.reddeer.common.wait.TimePeriod;
+import org.eclipse.reddeer.common.wait.WaitUntil;
+import org.eclipse.reddeer.common.wait.WaitWhile;
+import org.eclipse.reddeer.jface.wizard.WizardDialog;
+import org.eclipse.reddeer.swt.api.Button;
+import org.eclipse.reddeer.swt.condition.ShellIsActive;
+import org.eclipse.reddeer.swt.impl.browser.InternalBrowser;
+import org.eclipse.reddeer.swt.impl.button.CheckBox;
+import org.eclipse.reddeer.swt.impl.button.PushButton;
+import org.eclipse.reddeer.swt.impl.combo.DefaultCombo;
+import org.eclipse.reddeer.swt.impl.shell.DefaultShell;
+import org.eclipse.reddeer.swt.impl.text.LabeledText;
+import org.eclipse.reddeer.workbench.core.condition.JobIsRunning;
+import org.jboss.tools.central.reddeer.api.JavaScriptHelper;
+import org.jboss.tools.central.reddeer.projects.ArchetypeProject;
+import org.jboss.tools.central.reddeer.wait.CentralIsLoaded;
+import org.jboss.tools.central.reddeer.wizards.NewProjectExamplesReadyPage;
+
+
+
+/**
+ * 
+ * @author zcervink
+ *
+ */
+
+
+public class JBossCentralProjectWizardLauncherApplication extends WizardDialog {
+
+	protected final static Logger log = Logger
+			.getLogger(JBossCentralProjectWizardLauncherApplication.class);
+	private ArchetypeProject project;
+
+	public JBossCentralProjectWizardLauncherApplication(ArchetypeProject project) {
+		this.project = project;
+	}
+
+	public void open() {
+		new WaitUntil(new CentralIsLoaded(),TimePeriod.LONG);
+		JavaScriptHelper.getInstance().setBrowser(new InternalBrowser());
+		JavaScriptHelper.getInstance().clickWizard(project.getName());
+		new DefaultShell("New Launcher project");
+	}
+
+	/**
+	 * Finishes this wizard and returns {@link NewProjectExamplesReadyPage} for
+	 * user to validate that shell (for example for opening of cheatsheet).
+	 * 
+	 */
+
+	public NewProjectExamplesReadyPage finishAndWait() {
+		return finishAndWait(TimePeriod.getCustom(1800));
+	}
+
+	/**
+	 * Finishes this wizard with custom timeout and returns {@link NewProjectExamplesReadyPage} for
+	 * user to validate that shell (for example for opening of cheatsheet).
+	 * @param customTimeout
+	 */
+
+	public NewProjectExamplesReadyPage finishAndWait(TimePeriod customTimeout) {
+		log.info("Finish example wizard");
+
+		DefaultShell shell = new DefaultShell();
+		Button button = new PushButton("Finish");
+		button.click();
+		new WaitWhile(new ShellIsActive(shell), customTimeout);
+		new WaitWhile(new JobIsRunning(), TimePeriod.VERY_LONG);
+		return new NewProjectExamplesReadyPage(project);
+	}
+	
+	/**
+	 * Finishes this wizard without possibility to check
+	 * NewProjectExamplesReadyPage.
+	 */
+
```

Comment:
I would suggest to remove this empty line before this method and previous methods, because it helps with core readability (as you have this in next methods).

Reply:
removed